travis_test.sh

```diff
@@ -26,6 +26,7 @@ install_snappy
 echo "travis_fold:end:install.snappy"
 echo "travis_fold:start:install.pythondeps"
 pip install flake8
+pip install pytest==3.6.3
```

Comment:
do you want to make this 4?

Reply:
pytest 4 does not exist yet :) 3.6.3 is the latest version

I put in that `tests_require` restriction just based on the warnings' admonishment, not because anything broke yet. Thus, this PR does not actually fix anything, it just avoids deprecation warnings.